### ISSUE TITLE
2025-04-25, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#151]
# Today's Arxiv Papers

(1) [Token-Shuffle: Towards High-Resolution Image Generation with Autoregressive Models](http://arxiv.org/abs/2504.17789v1), 2025-04-24

(2) [Replay to Remember: Retaining Domain Knowledge in Streaming Language Models](http://arxiv.org/abs/2504.17780v1), 2025-04-24

(3) [The Sparse Frontier: Sparse Attention Trade-offs in Transformer LLMs](http://arxiv.org/abs/2504.17768v1), 2025-04-24

(4) [Step1X-Edit: A Practical Framework for General Image Editing](http://arxiv.org/abs/2504.17761v1), 2025-04-24

(5) [Conversational Assistants to support Heart Failure Patients: comparing a Neurosymbolic Architecture with ChatGPT](http://arxiv.org/abs/2504.17753v1), 2025-04-24

(6) [Towards Robust LLMs: an Adversarial Robustness Measurement Framework](http://arxiv.org/abs/2504.17723v1), 2025-04-24

(7) [Multilingual Performance Biases of Large Language Models in Education](http://arxiv.org/abs/2504.17720v1), 2025-04-24

(8) [Ensemble Bayesian Inference: Leveraging Small Language Models to Achieve LLM-level Accuracy in Profile Matching Tasks](http://arxiv.org/abs/2504.17685v1), 2025-04-24

(9) [INSIGHT: Bridging the Student-Teacher Gap in Times of Large Language Models](http://arxiv.org/abs/2504.17677v1), 2025-04-24

(10) [Energy Considerations of Large Language Model Inference and Efficiency Optimizations](http://arxiv.org/abs/2504.17674v1), 2025-04-24

(11) [Cross-region Model Training with Communication-Computation Overlapping and Delay Compensation](http://arxiv.org/abs/2504.17672v1), 2025-04-24

(12) [Towards a HIPAA Compliant Agentic AI System in Healthcare](http://arxiv.org/abs/2504.17669v1), 2025-04-24

(13) [Evaluating Grounded Reasoning by Code-Assisted Large Language Models for Mathematics](http://arxiv.org/abs/2504.17665v1), 2025-04-24

(14) [L3: DIMM-PIM Integrated Architecture and Coordination for Scalable Long-Context LLM Inference](http://arxiv.org/abs/2504.17584v1), 2025-04-24

(15) [DeepDistill: Enhancing LLM Reasoning Capabilities via Large-Scale Difficulty-Graded Data Training](http://arxiv.org/abs/2504.17565v1), 2025-04-24

(16) [HalluLens: LLM Hallucination Benchmark](http://arxiv.org/abs/2504.17550v1), 2025-04-24

(17) [A Comprehensive Survey of Knowledge-Based Vision Question Answering Systems: The Lifecycle of Knowledge in Visual Reasoning Task](http://arxiv.org/abs/2504.17547v1), 2025-04-24

(18) [Auditing the Ethical Logic of Generative AI Models](http://arxiv.org/abs/2504.17544v1), 2025-04-24

(19) [Large Language Model-Driven Concolic Execution for Highly Structured Test Input Generation](http://arxiv.org/abs/2504.17542v1), 2025-04-24

(20) [Towards Machine-Generated Code for the Resolution of User Intentions](http://arxiv.org/abs/2504.17531v1), 2025-04-24

(21) [Combining GCN Structural Learning with LLM Chemical Knowledge for or Enhanced Virtual Screening](http://arxiv.org/abs/2504.17497v1), 2025-04-24

(22) [Unified Attacks to Large Language Model Watermarks: Spoofing and Scrubbing in Unauthorized Knowledge Distillation](http://arxiv.org/abs/2504.17480v1), 2025-04-24

(23) [Adaptive Orchestration of Modular Generative Information Access Systems](http://dx.doi.org/10.1145/3726302.3730351), 2025-04-24

(24) [Creating Targeted, Interpretable Topic Models with LLM-Generated Text Augmentation](http://arxiv.org/abs/2504.17445v1), 2025-04-24

(25) [Breaking the Modality Barrier: Universal Embedding Learning with Multimodal LLMs](http://arxiv.org/abs/2504.17432v1), 2025-04-24

(26) [Beyond Whole Dialogue Modeling: Contextual Disentanglement for Conversational Recommendation](http://arxiv.org/abs/2504.17427v1), 2025-04-24

(27) [Towards Leveraging Large Language Model Summaries for Topic Modeling in Source Code](http://arxiv.org/abs/2504.17426v1), 2025-04-24

(28) [Towards Harnessing the Collaborative Power of Large and Small Models for Domain Tasks](http://arxiv.org/abs/2504.17421v1), 2025-04-24

(29) [Assessing the Capability of Large Language Models for Domain-Specific Ontology Generation](http://arxiv.org/abs/2504.17402v1), 2025-04-24

(30) [On-Device Qwen2.5: Efficient LLM Inference with Model Compression and Hardware Acceleration](http://arxiv.org/abs/2504.17376v1), 2025-04-24

(31) [LiveLongBench: Tackling Long-Context Understanding for Spoken Texts from Live Streams](http://arxiv.org/abs/2504.17366v1), 2025-04-24

(32) [TimeSoccer: An End-to-End Multimodal Large Language Model for Soccer Commentary Generation](http://arxiv.org/abs/2504.17365v1), 2025-04-24

(33) [PatientDx: Merging Large Language Models for Protecting Data-Privacy in Healthcare](http://arxiv.org/abs/2504.17360v1), 2025-04-24

(34) [Comprehend, Divide, and Conquer: Feature Subspace Exploration via Multi-Agent Hierarchical Reinforcement Learning](http://arxiv.org/abs/2504.17356v1), 2025-04-24

(35) [DRC: Enhancing Personalized Image Generation via Disentangled Representation Composition](http://arxiv.org/abs/2504.17349v1), 2025-04-24

(36) [TimeChat-Online: 80% Visual Tokens are Naturally Redundant in Streaming Videos](http://arxiv.org/abs/2504.17343v1), 2025-04-24

(37) [DataScout: Automatic Data Fact Retrieval for Statement Augmentation with an LLM-Based Agent](http://arxiv.org/abs/2504.17334v1), 2025-04-24

(38) [Bridging Cognition and Emotion: Empathy-Driven Multimodal Misinformation Detection](http://arxiv.org/abs/2504.17332v1), 2025-04-24

(39) [Exploring Context-aware and LLM-driven Locomotion for Immersive Virtual Reality](http://arxiv.org/abs/2504.17331v1), 2025-04-24

(40) [FLUKE: A Linguistically-Driven and Task-Agnostic Framework for Robustness Evaluation](http://arxiv.org/abs/2504.17311v1), 2025-04-24

(41) [CoheMark: A Novel Sentence-Level Watermark for Enhanced Text Quality](http://arxiv.org/abs/2504.17309v1), 2025-04-24

(42) [You Are What You Bought: Generating Customer Personas for E-commerce Applications](http://dx.doi.org/10.1145/3726302.3730118), 2025-04-24

(43) [AI-Enhanced Business Process Automation: A Case Study in the Insurance Domain Using Object-Centric Process Mining](http://arxiv.org/abs/2504.17295v1), 2025-04-24

(44) [Combining Static and Dynamic Approaches for Mining and Testing Constraints for RESTful API Testing](http://arxiv.org/abs/2504.17287v1), 2025-04-24

(45) [MV-Crafter: An Intelligent System for Music-guided Video Generation](http://arxiv.org/abs/2504.17267v1), 2025-04-24

(46) [JurisCTC: Enhancing Legal Judgment Prediction via Cross-Domain Transfer and Contrastive Learning](http://arxiv.org/abs/2504.17264v1), 2025-04-24

(47) [Crisp: Cognitive Restructuring of Negative Thoughts through Multi-turn Supportive Dialogues](http://arxiv.org/abs/2504.17238v1), 2025-04-24

(48) [FLAG: Formal and LLM-assisted SVA Generation for Formal Specifications of On-Chip Communication Protocols](http://arxiv.org/abs/2504.17226v1), 2025-04-24

(49) [Visual and textual prompts for enhancing emotion recognition in video](http://arxiv.org/abs/2504.17224v1), 2025-04-24

(50) [Does Knowledge Distillation Matter for Large Language Model based Bundle Generation?](http://arxiv.org/abs/2504.17220v1), 2025-04-24

(51) [High-Fidelity And Complex Test Data Generation For Real-World SQL Code Generation Services](http://arxiv.org/abs/2504.17203v1), 2025-04-24

(52) [A RAG-Based Multi-Agent LLM System for Natural Hazard Resilience and Adaptation](http://arxiv.org/abs/2504.17200v1), 2025-04-24

(53) [Automatically Generating Rules of Malicious Software Packages via Large Language Model](http://arxiv.org/abs/2504.17198v1), 2025-04-24

(54) [Paper2Code: Automating Code Generation from Scientific Papers in Machine Learning](http://arxiv.org/abs/2504.17192v1), 2025-04-24

(55) [MIRAGE: A Metric-Intensive Benchmark for Retrieval-Augmented Generation Evaluation](http://arxiv.org/abs/2504.17137v1), 2025-04-23

(56) [Steering the CensorShip: Uncovering Representation Vectors for LLM "Thought" Control](http://arxiv.org/abs/2504.17130v1), 2025-04-23

(57) [The Rise of Small Language Models in Healthcare: A Comprehensive Survey](http://arxiv.org/abs/2504.17119v1), 2025-04-23

(58) [Leveraging LLMs as Meta-Judges: A Multi-Agent Framework for Evaluating LLM Judgments](http://arxiv.org/abs/2504.17087v1), 2025-04-23

(59) [How Individual Traits and Language Styles Shape Preferences In Open-ended User-LLM Interaction: A Preliminary Study](http://arxiv.org/abs/2504.17083v1), 2025-04-23

(60) [Agree to Disagree? A Meta-Evaluation of LLM Misgendering](http://arxiv.org/abs/2504.17075v1), 2025-04-23

(61) [Robo-Troj: Attacking LLM-based Task Planners](http://arxiv.org/abs/2504.17070v1), 2025-04-23

(62) [Do Words Reflect Beliefs? Evaluating Belief Depth in Large Language Models](http://arxiv.org/abs/2504.17052v1), 2025-04-23

(63) [DyMU: Dynamic Merging and Virtual Unmerging for Efficient VLMs](http://arxiv.org/abs/2504.17040v1), 2025-04-23

(64) [Optimizing LLMs for Italian: Reducing Token Fertility and Enhancing Efficiency Through Vocabulary Adaptation](http://arxiv.org/abs/2504.17025v1), 2025-04-23

(65) [LLM impact on BLV programming](http://arxiv.org/abs/2504.17018v1), 2025-04-23

(66) [Neural Theorem Proving: Generating and Structuring Proofs for Formal Verification](http://arxiv.org/abs/2504.17017v1), 2025-04-23

(67) [(Im)possibility of Automated Hallucination Detection in Large Language Models](http://arxiv.org/abs/2504.17004v1), 2025-04-23

(68) [Safety Pretraining: Toward the Next Generation of Safe AI](http://arxiv.org/abs/2504.16980v1), 2025-04-23

(69) [IberBench: LLM Evaluation on Iberian Languages](http://arxiv.org/abs/2504.16921v1), 2025-04-23

(70) [OptimAI: Optimization from Natural Language Using LLM-Powered AI Agents](http://arxiv.org/abs/2504.16918v1), 2025-04-23

(71) [Tracing Thought: Using Chain-of-Thought Reasoning to Identify the LLM Behind AI-Generated Text](http://arxiv.org/abs/2504.16913v1), 2025-04-23

(72) [Do Large Language Models know who did what to whom?](http://arxiv.org/abs/2504.16884v1), 2025-04-23

(73) [Enhancing Critical Thinking with AI: A Tailored Warning System for RAG Models](http://arxiv.org/abs/2504.16883v1), 2025-04-23

(74) [Context-Enhanced Vulnerability Detection Based on Large Language Model](http://arxiv.org/abs/2504.16877v1), 2025-04-23

(75) [Exploring How LLMs Capture and Represent Domain-Specific Knowledge](http://arxiv.org/abs/2504.16871v2), 2025-04-23

(76) [Planning with Diffusion Models for Target-Oriented Dialogue Systems](http://arxiv.org/abs/2504.16858v1), 2025-04-23

(77) [Emo Pillars: Knowledge Distillation to Support Fine-Grained Context-Aware and Context-Less Emotion Classification](http://arxiv.org/abs/2504.16856v1), 2025-04-23

(78) [Monte Carlo Planning with Large Language Model for Text-Based Game Agents](http://arxiv.org/abs/2504.16855v1), 2025-04-23

(79) [Improving Significant Wave Height Prediction Using Chronos Models](http://arxiv.org/abs/2504.16834v1), 2025-04-23

(80) [LRASGen: LLM-based RESTful API Specification Generation](http://arxiv.org/abs/2504.16833v1), 2025-04-23

(81) [GreenMind: A Next-Generation Vietnamese Large Language Model for Structured and Logical Reasoning](http://arxiv.org/abs/2504.16832v1), 2025-04-23

(82) [Process Reward Models That Think](http://arxiv.org/abs/2504.16828v1), 2025-04-23

(83) [LLM-assisted Graph-RAG Information Extraction from IFC Data](http://arxiv.org/abs/2504.16813v1), 2025-04-23

(84) [Decoupled Global-Local Alignment for Improving Compositional Understanding](http://arxiv.org/abs/2504.16801v1), 2025-04-23

(85) [MOOSComp: Improving Lightweight Long-Context Compressor via Mitigating Over-Smoothing and Incorporating Outlier Scores](http://arxiv.org/abs/2504.16786v1), 2025-04-23

(86) [Graph2Nav: 3D Object-Relation Graph Generation to Robot Navigation](http://arxiv.org/abs/2504.16782v1), 2025-04-23

(87) [How Effective are Generative Large Language Models in Performing Requirements Classification?](http://arxiv.org/abs/2504.16768v1), 2025-04-23

(88) [Lightweight Latent Verifiers for Efficient Meta-Generation Strategies](http://arxiv.org/abs/2504.16760v1), 2025-04-23

(89) [HEMA : A Hippocampus-Inspired Extended Memory Architecture for Long-Context AI Conversations](http://arxiv.org/abs/2504.16754v1), 2025-04-23

(90) [A Survey of AI Agent Protocols](http://arxiv.org/abs/2504.16736v1), 2025-04-23

(91) [IRIS: Interactive Research Ideation System for Accelerating Scientific Discovery](http://arxiv.org/abs/2504.16728v1), 2025-04-23

(92) [A Post-trainer's Guide to Multilingual Training Data: Uncovering Cross-lingual Transfer Dynamics](http://arxiv.org/abs/2504.16677v1), 2025-04-23

(93) [LLMCode: Evaluating and Enhancing Researcher-AI Alignment in Qualitative Analysis](http://arxiv.org/abs/2504.16671v1), 2025-04-23

(94) [Backslash: Rate Constrained Optimized Training of Large Language Models](http://arxiv.org/abs/2504.16968v1), 2025-04-23

(95) [ParetoHqD: Fast Offline Multiobjective Alignment of Large Language Models using Pareto High-quality Data](http://arxiv.org/abs/2504.16628v1), 2025-04-23

(96) [TIFIN India at SemEval-2025: Harnessing Translation to Overcome Multilingual IR Challenges in Fact-Checked Claim Retrieval](http://arxiv.org/abs/2504.16627v1), 2025-04-23

(97) [Cognitive Silicon: An Architectural Blueprint for Post-Industrial Computing Systems](http://arxiv.org/abs/2504.16622v1), 2025-04-23

(98) [Algorithmic Mirror: Designing an Interactive Tool to Promote Self-Reflection for YouTube Recommendations](http://arxiv.org/abs/2504.16615v1), 2025-04-23

(99) [Debunking with Dialogue? Exploring AI-Generated Counterspeech to Challenge Conspiracy Theories](http://arxiv.org/abs/2504.16604v1), 2025-04-23

(100) [Comparing Large Language Models and Traditional Machine Translation Tools for Translating Medical Consultation Summaries: A Pilot Study](http://arxiv.org/abs/2504.16601v1), 2025-04-23
